docs/releaseNotes.yml

```diff
@@ -34,6 +34,14 @@ items:
   - version: 2.1.0
     date: 'TBD'
     notes:
+      - title: CRD conversion webhooks
+        type: feature
+        body: >-
+          The ambassador-admin service now hosts a webhook server that enables the
+          user to seamlessly request CRs in whatever API version is desired. The
+          webhook server leverages its own CA to perform the TLS connections required
+          of it. The CA certificate and private key are stored in a k8s secret.
```

Comment:
This isn't a feedback, but just a stupid curiosity question: This text is wrapped to about 85 columns, but the top of the file says `# -*- fill-column: 100 -*-` and you use Emacs. Are you wrapping the text by hand instead of using M-q?  Is your Emacs ignoring the `-*-` line for some reason?

Reply:
I wrote that out by hand.